docs/next.config.mjs

```diff
@@ -13,4 +13,7 @@ const withNextra = nextra({
 export default withNextra({
   reactStrictMode: true,
   eslint: { ignoreDuringBuilds: true },
+  env: {
+    NO_INDEX: process.env.VERCEL_ENV !== "production" ? "true" : "false",
+  },
```

Comment:
need `nextConfig.env` to inject an otherwise server-only env to the client bundle